src/Castle.Windsor/MicroKernel/SubSystems/Conversion/ComponentModelConverter.cs

```diff
@@ -29,12 +29,6 @@ public class ComponentModelConverter : AbstractTypeConverter
 	{
 		public override bool CanHandleType(Type type)
 		{
-			// Mono 1.9+ thinks it can convert strings to interface
-			if (type.IsInterface)
```

Comment:
This test doesn't seem like a Mono only check. I assume you can't have a converter for an interface so doesn't this just speed up this method without the need to call `TypeDescriptor.GetConverter` which will always return null for interfaces?